### DISCUSSION TITLE
IKEA SYMFONISK Sound Controller - How to expose simulated brightness?

### DISCUSSION THREAD

**Discussion by sabaatworld** [Koenkk/zigbee2mqtt#11334] [Devices]
Device: https://www.zigbee2mqtt.io/devices/E1744.html#ikea-e1744

After setting up simulated brightness, I'm able to see the value for brightness in "state" as follows:

```
{
    "battery": 47,
    "update": {
        "state": "idle"
    },
    "action": "brightness_stop",
    "action_rate": 195,
    "linkquality": 36,
    "brightness": 198
}
```

How can I add this to "Exposes" so that its visible to Homebridge using homebridge-z2m?

**Comment by pabutterworth** [comment]
How do I get to see this brightness value?
I'm struggling to setup the Symfonisk as an effective volume control, as I seem to have no idea how much the device has been turned.

**Reply by sabaatworld** [reply]
Don't try to set it up manually. Simply use controllerX

https://xaviml.github.io/controllerx/

**Comment by pabutterworth** [comment]
Thanks, am looking at that, but am struggling to understand what I need to
do. I have installed the app daemon and ControllerX, but can't work out
what settings to use. I'm using Zigbee2MQTT, an Ikea E1744LightController
to control an Ikea Symfonisk speaker.

Any tips are appreciated.

Thanks

On Wed, 9 Mar 2022 at 18:13, Sabaat Ahmad ***@***.***> wrote:

> Don't try to set it up manually. Simply use controllerX
>
> https://xaviml.github.io/controllerx/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Koenkk/zigbee2mqtt/discussions/11334#discussioncomment-2326140>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD737V4V5X5XMF43JCW6R7LU7DS43ANCNFSM5N6NXSVQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Paul Butterworth
+44 7768 252250

**Reply by pabutterworth** [reply]
I used this setup:

sonos_control:
  module: controllerx
  class: E1744MediaPlayerController
  controller: sonos_remote
  integration: z2m
  media_player: kitchen

Anything obvious?

**Comment by sabaatworld** [comment]
Both controller and media player should be entity ids and not just names of the devices in home assistant.

Also everything except for the first line should be indented by 2 spaces.

**Comment by pabutterworth** [comment]
Awesome, working now, many thanks for your help.

On Thu, 10 Mar 2022 at 14:09, Sabaat Ahmad ***@***.***> wrote:

> Both controller and media player should be entity ids and not just names
> of the devices in home assistant.
>
> Also everything except for the first line should be indented by 2 spaces.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Koenkk/zigbee2mqtt/discussions/11334#discussioncomment-2331782>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD737V7UMHMPPDSYPEQBISLU7H7BHANCNFSM5N6NXSVQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Paul Butterworth
+44 7768 252250
